rllib/core/rl_trainer/rl_trainer.py

```diff
@@ -83,62 +86,110 @@ class RLTrainerHPs:
 
 
 class RLTrainer:
-    """Base class for RLlib algorithm trainers.
+    """Base class for learners.
```

Comment:
I decided to go with learner for the docstrings, because it will be harder to fix them later.